src/material/datepicker/calendar-body.ts

```diff
@@ -159,18 +165,31 @@ export class MatCalendarBody implements OnChanges, OnDestroy, AfterViewChecked {
   /** Width of an individual cell. */
   _cellWidth: string;
 
+  private _didDragSinceMouseDown = false;
+
```

Comment:
Unrelated to this line, but I noticed that sometimes the highlight was being cleared when I stop dragging. See the cell for 19 below.
<img width="238" alt="image" src="https://user-images.githubusercontent.com/4450522/197730880-200b3b41-d7e0-4ec4-917e-8958b84d0d99.png">

Reply:
Not sure what you mean - the 19 appears to be in the currently hovered state. Is the mouse not currently over 19?

Reply:
I wasn't hovering the 19, I started dragging from it but my mouse was completely outside of the calendar.

Reply:
I couldn't really track this one down. One thing I noticed is that the hover effect does not show up in chrome's simulated touch mode. It seems to work as expected in mouse mode the vast majority of the time. Are we willing to live with this for now?

Reply:
Sure, we can fix it in a follow-up.